src/components/Class.vue

```diff
@@ -99,9 +104,32 @@ export default {
   data () {
     return {
       warningDialog: false,
-      shouldOverrideWarnings: this.classInfo.overrideWarnings
+      shouldOverrideWarnings: this.classInfo.overrideWarnings,
+      useShortenedTitle: false
```

Comment:
Without context this seems like a global property, could you add a comment that says this will be changed to true by VueLineClamp if the title is too long?